rpm/tarantool-checks.spec

```diff
@@ -22,10 +22,13 @@ Easy, terse, readable and fast function arguments type checking.
 mkdir -p %{br_luapkgdir}
 cp -av checks.lua %{br_luapkgdir}
 cp -rv checks %{br_luapkgdir}
+cp -av checks.lua %{br_luapkgdir}/override
+cp -rv checks %{br_luapkgdir}/override
 
 %files
 %{luapkgdir}/checks.lua
 %{luapkgdir}/checks
+%{luapkgdir}/override
```

Comment:
How about the following?

```
%{luapkgdir}/override/checks.lua
%{luapkgdir}/override/checks
```

I don't know, to be honest, whether it makes any difference, but it seems logical to advertize ownership about files/directories we actually own (where nobody else installs anything).

Reply:
Reworked